App.js

```diff
@@ -1,72 +1,16 @@
 import React from 'react';
```

Comment:
Make sure this file matches the `master` branch verison. I've said this too many times, make sure that your PRs thoroughly.